### PR TITLE
Finished homework

### PR DESCRIPTION
**Pull request by dan-nil**

### CODE REVIEW

File: react/react1/week1/todo-app/src/components/ListTodos.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Todo from './Todo';
+
+export default function ListToDos({ todos }) {
+    return (
+        <ul className='todos-container'>
+            {todos.map(todo => (
+                <Todo title={todo.title} date={todo.date} />
```
Comment (annamourasobral) [inline react/react1/week1/todo-app/src/components/ListTodos.js]: You could pass `todo={todo}` and destructure its props inside `Todo.jsx` :)

**Review by annamourasobral — APPROVED** [review]

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey @danJecu 
Good job!
i miss clicked approve

File: react/react1/week1/todo-app/src/components/ListTodos.js (hunk 1 above)
Comment (dan-nil) [reply react/react1/week1/todo-app/src/components/ListTodos.js]: I changed that :)

Comment (dan-nil) [reply react/react1/week1/todo-app/src/components/ListTodos.js]: Thank you for the review!

**Pull request merged**